### PR TITLE
Feature: social logins

### PR DESCRIPTION
**Pull request by YogeshRst**
here we add additional details for contributor...

pull requests are created if you want a second opinion from contributors. or if you just want them to check the code or review it 
before merge

### CODE REVIEW

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (level1cr3) [inline file2.txt]: T should be capital

**Review by level1cr3 — CHANGES_REQUESTED** [review]
General comments  based on review

File: file2.txt (hunk 1 above)
Comment (YogeshRst) [reply file2.txt]: Ok will do that

**Review by YogeshRst — COMMENTED** [review]
some general text

**Comment by level1cr3** [comment]
Ya ok do that

**Review by level1cr3 — APPROVED** [review]
Ya that looks good you can go ahead with merge

**Pull request merged**